boards/shields/mcp2515/keyestudio_can_bus_ks0411.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Henrik Brix Andersen <henrik@brixandersen.dk>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>; /* D10 */
+
+	mcp2515: can@0 {
+		compatible = "microchip,mcp2515";
+		spi-max-frequency = <1000000>;
+		int-gpios = <&arduino_header 14 GPIO_ACTIVE_LOW>; /* D8 */
+		status = "okay";
+		label = "MCP2515";
+		reg = <0x0>;
+		osc-freq = <16000000>;
+		bus-speed = <125000>;
+		sjw = <1>;
+		prop-seg = <2>;
+		phase-seg1 = <7>;
```

Comment:
Shouldn't we use the sample point instead?

Reply:
Yes. I intend to do a full sweep of the tree and replace it all over. I have changed it for the MCP2515 shields now.